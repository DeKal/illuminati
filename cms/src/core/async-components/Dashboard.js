import loadable from '@loadable/component'

const Dashboard = loadable(() =>
  import(/* webpackChunkName: "post" */ 'dashboard/components/Dashboard')
)

export default Dashboard
