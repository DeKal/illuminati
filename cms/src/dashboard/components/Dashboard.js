import React from 'react'

const DashBoard = () => (
  <div>
    <h2> Dashboard</h2>
  </div>
)

export default DashBoard
