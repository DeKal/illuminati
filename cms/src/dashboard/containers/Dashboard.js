import Dashboard from 'dashboard/components/Dashboard'
import { withRouter } from 'react-router'

const DashBoardWithRouter = withRouter(Dashboard)

export default DashBoardWithRouter
