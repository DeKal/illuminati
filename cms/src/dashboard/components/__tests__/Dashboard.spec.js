import * as React from 'react'
import { shallow } from 'enzyme'
import Dashboard from '../Dashboard'

describe('Dashboard', () => {
  it('should render Dashboard with no error', function() {
    const wrap = shallow(<Dashboard />)
    expect(wrap).toMatchSnapshot()
  })
})
