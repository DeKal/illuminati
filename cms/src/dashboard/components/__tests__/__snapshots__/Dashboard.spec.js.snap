// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should render Dashboard with no error 1`] = `
<div>
  <h2>
     Dashboard
  </h2>
</div>
`;
